front/src/tabletview/rangeofficer.js

```diff
@@ -482,7 +482,10 @@ const Tabletview = () => {
     const res = await rangeSupervision(reservationId,
       scheduleId,
       status,
-      rangeSupervisionScheduled);
+
+      rangeSupervisionScheduled,
+      null); // MODIFY THIS
+
```

Comment:
modify how? Is this comment necessary?

Reply:
Probably not anymore as null handling has been done. Can be removed.

Reply:
removed